data-raw/dm_metabolic.R

```diff
@@ -8,7 +8,7 @@ library(dplyr)
 library(purrr)
 
 # Import DM data from `pharmaversesdtm` ----
-data("dm")
+pharmaversesdtm::dm
```

Comment:
```suggestion
dm <- pharmaversesdtm::dm
```

The reason for this change is because we want the object we are using further down in the function to be defined in the global environment. 

The code still works without it, but we would be beholden to the load order of packages (scoping) if there are several packages that provide `dm` data.

Reply:
I will update it asap, thank you for the review.